### PR TITLE
WIP - Add doc placeholder for vSphere CSI Migration

### PR DESCRIPTION
**Pull request by xing-yang**
This is a placeholder doc for vSphere CSI Migration.

### CODE REVIEW

File: content/en/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -1118,6 +1118,17 @@ spec:
 
 More examples can be found [here](https://github.com/kubernetes/examples/tree/master/staging/volumes/vsphere).
 
+#### CSI Migration
+
+{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
+
+The CSI Migration feature for vsphereVolume, when enabled, shims all plugin operations
+from the existing in-tree plugin to the `csi.vsphere.vmware.com` Container
+Storage Interface (CSI) Driver. In order to use this feature, the [vSphere CSI
```
Comment (sftim) [inline content/en/docs/concepts/storage/volumes.md]: I'd consider using `{{< glossary_tooltip text="Container Storage Interface" term_id="csi" >}}` to get a glossary tooltip and hyperlink.

**Comment by davidz627** [comment]
This is no longer targeted for `1.18` IIUC. Feel free to close and open another for the next release when it is time

**Comment by xing-yang** [comment]
Sure.  Closing it now.

**Pull request closed without merging**